app/views/assessed_claims/_assessed_claims_table.html.erb

```diff
@@ -0,0 +1,38 @@
+<%=
+    rows = claims.map do |claim|
+      [*claim.table_fields]
+    end
+
+    govuk_table(html_attributes: {'data-module': 'moj-sortable-table'}) do |table|
+      table.with_caption(html_attributes: { class: "govuk-visually-hidden" }, text: t(".table_heading"))
+
+      table.with_head do |head|
+        head.with_row do |row|
+          row.with_cell(text: t('.laa_reference'))
+          row.with_cell(text: t('.firm_name'))
+          row.with_cell(text: t('.defendant_name'))
+          row.with_cell(html_attributes:{'aria-sort': 'ascending'},text: t('.date_assessed'))
+          row.with_cell(html_attributes:{'aria-sort': 'ascending'},text: t('.caseworker_name'))
+          row.with_cell(html_attributes:{'aria-sort': 'ascending'},text: t('.status'))
+        end
+      end
+
+      table.with_body do |body|
+        rows.each do |item|
```

Comment:
so you can make this easier by instead doing:
```
rows.each do |(id, firm, defendant, assessed, case_worker, status)|
```
not 100% sure you need the round brackets

Reply:
Will change it to next PR. Thanks